gestalt/__init__.py

```diff
@@ -549,7 +549,7 @@ def _get_config_for_key(
         self, key: str, key_to_search: str,
         default: Optional[Union[str, int, float, bool, List[Any]]],
         object_type: Type[Union[str, int, float, bool, List[Any]]]
-    ) -> Optional[Union[str, int, float, bool, List[Any]]]:
+    ) -> Optional[Union[str, int, float, bool, List[Any]]]:  # noqa
```

Comment:
We probably shouldn't bare except `noqa`

Reply:
removed the `# noqa` , interestingly it passes now 🤔